net.lutris.Lutris.yml

```diff
@@ -0,0 +1,205 @@
+id: net.lutris.Lutris
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "3.32"
+command: lutris
+rename-icon: lutris
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --allow=multiarch
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.Mutter.DisplayConfig
+  - --talk-name=ca.desrt.dconf
+  - --filesystem=xdg-run/dconf
+  - --filesystem=xdg-config/dconf:ro
+  - --filesystem=~/Games:create
+  - --filesystem=xdg-data/Steam:ro
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:ro
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: "18.08"
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: "18.08"
+    no-autodownload: true
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: "1.4"
+    versions: "18.08;1.4"
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /share/man
+  - /share/help
+cleanup-commands:
+  - python3 -m compileall /app/lib
+modules:
+
+  - name: lutris
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    post-install:
+      - desktop-file-edit --set-key=X-Flatpak-RenamedFrom --set-value="lutris.desktop;" /app/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: git
+        url: "https://github.com/lutris/lutris.git"
+        tag: v0.5.1.2
+      - type: patch
+        path: patches/0001-Add-flatpak-ed-Steam-data-dir.patch
+      - type: patch
+        path: patches/0002-Use-xdg-open-to-run-Steam-games-if-running-in-flatpa.patch
+    modules:
+
+      - name: gnome-desktop
+        buildsystem: meson
+        sources:
+          - type: archive
+            url: "https://download.gnome.org/sources/gnome-desktop/3.32/gnome-desktop-3.32.0.tar.xz"
+            sha256: a6393dc5fc29fc0652ac84c73b3da205d0b0168128c4cf6d27797a08f3d07b54
+
+      - name: python-evdev
+        buildsystem: simple
+        build-commands:
+          - python3 setup.py install --prefix=/app --root=/
+        sources:
+          - type: archive
+            url: "https://github.com/gvalkov/python-evdev/archive/v1.1.2.tar.gz"
+            sha256: 4403d4ee2d9682d17d292d5cfe0781d9ebfe119c7bb0f631fb59de3003bad49d
+
+      - name: PyYAML
+        buildsystem: simple
+        build-commands:
+          - python3 setup.py install --prefix=/app --root=/
```

Comment:
[flatpak-builder-tools/pip](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip) generates the following command to install PyYAML:

```
pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} PyYAML
```

Is there any case where prefix wouldn't be /app?

Reply:
AFAIK only in case of building a runtime or extension.